content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -68,47 +68,16 @@ The DEKs are encrypted with a key encryption key (KEK) that is stored and manage
 
 With KMS v1, a new DEK is generated for each encryption.
 
```

Comment:
```suggestion
If you use the (deprecated) v1 implementation of KMS, a new DEK is generated for each encryption.
```

Reply:
We might not need to mention the difference, as (AIUI) both v1 and v2 generate a DEK per encryption.